src/main/java/frc/robot/components/MotorConfig.java

```diff
@@ -2,25 +2,27 @@
 
 import com.ctre.phoenix.motorcontrol.NeutralMode;
 import com.ctre.phoenix.motorcontrol.SupplyCurrentLimitConfiguration;
+import frc.robot.utilities.PIDCoefs;
 
 /**
  * This class stores the configuration of a motor
  */
-public class MotorConfig {
+public class MotorConfig{
 
     private double rampRate;
     private boolean isInverted;
     private boolean isSensorInverted;
     private NeutralMode neutralMode;
     private double voltageCompSaturation;
     private SupplyCurrentLimitConfiguration currentLimitConfig;
+    private PIDCoefs coefs;
```

Comment:
i feel like PIDCoefs is too specific and this class is way too jampacked already. also you anyways have a pidCoefs instance saved in the constants. unlike inverted and ramprate which you don't save separately. If you want to automate it add it in the TrigonTalon

Reply:
I think that the coefs should go here. look at where they go...
they go directly into the configuration of the motor

Reply:
but that means you either add 500 constructors or you have to give ALL the values everytime you want to add pid. I suggested a diffrent way to do it in a diffrent commit.

Reply:
I don't like having so many constructors. just put in `null` where you don't have a value for it...

Reply:
no

Reply:
I disagree. it should not go in this class

Reply:
Sorry, I don't see the problem. @lycaon7?